gr-blocks/lib/rotator_cc_impl.cc

```diff
@@ -40,9 +40,17 @@ rotator_cc::sptr rotator_cc::make(double phase_inc)
 rotator_cc_impl::rotator_cc_impl(double phase_inc)
     : sync_block("rotator_cc",
                  io_signature::make(1, 1, sizeof(gr_complex)),
-                 io_signature::make(1, 1, sizeof(gr_complex)))
+                 io_signature::make(1, 1, sizeof(gr_complex))),
+    d_idx_next_inc_update(0),
+    d_next_phase_inc(0.0),
+    d_inc_update_pending(false)
 {
     set_phase_inc(phase_inc);
+
+    message_port_register_in(pmt::mp("phase_inc"));
```

Comment:
We should make it a habit to not hard code the port name down here. Can you make a const further up?

Reply:
Agreed. Updated.